src/member.rs

```diff
@@ -299,8 +357,12 @@ where
 
                 event = self.resolved_requests.next() => match event {
                     Some(request) => match request {
-                        Request::RequestCoord(coord) => { self.not_resolved_coords.remove(&coord); },
-                        Request::RequestParents(u_hash) => { self.not_resolved_parents.remove(&u_hash); },
+                        Request::Coord(coord) => { self.not_resolved_coords.remove(&coord); },
+                        Request::Parents(u_hash) => { self.not_resolved_parents.remove(&u_hash); },
+                        Request::NewestUnit(_salt) => {
+                            // TODO: ensure that the salt is right?
```

Comment:
I think the point is that, it is impossible that the salt is incorrect, that's why the question mark. Because in one run, we use one value of salt and accept only responses with this particular salt. I think keeping it this way should be fine...

Reply:
Makes sense, although then I would prefer `_` to `_salt` and remove this TODO.

Reply:
Removed :>